tests/test_prompts.py

```diff
@@ -282,12 +282,191 @@ def test_generic_math_prompt_code_examples():
 # we will solve for b and c
 solutions = sp.solve((point_1,point_2), (b, c))
 solutions[c]
-</llm-code><llm-code-output>
+</llm-code>
```

Comment:
Can you please add one more test for qwen format? Good to have them explicit somewhere so that we can use tests as a quick reference of how the code tokens/output looks like